### ISSUE TITLE
Post SIT-3 Task (sds-data-manager)

### ISSUE THREAD

**Issue by tech3371** [IMAP-Science-Operations-Center/sds-data-manager#300]
These will be broken into individual issue:

- [x]  Create AWS lambda layer to minimize lambda code package size
- [x] Improve dependency association. Narrow down time association.
- [x] Add ways to check for calibration files dependency.
- [x] Implement data version tracking
- [ ] Improve/Implement database backup and duplicate record checks, sync in create_schema stack - Ask Laura/Gavin for information
- [x] Add check to avoid kicking off duplicate batch job when two upstream dependency files are ingested which will only need to kickoff one downstream job.
- [x] Clean up imap_cli command line inputs. Define what’s needed and what’s not needed. Make update to batch starter and indexer lambda as needed based on this change. And line up with `imap_processing` changes.
- [x] Add timestamp to status tracking table. Some way to record time of when some processing was done.
- [x] Updates to tools to build and push container. Discuss how to track release version in ECR using ECR tag and GitHub release tag. Look at this PR - https://github.com/IMAP-Science-Operations-Center/sds-data-manager/pull/297

**Comment by greglucas** [comment]
@tech3371, Are we able to close more of these issues, do we have plans for all of the sub-tasks already?

**Comment by tech3371** [comment]
Yep. We can close that and there is sub-task issue for that database migration.
